src/lib/firestore/CollectionAggregateState.svelte.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  type Query,
+  type AggregateSpec,
+  getAggregateFromServer,
+  collection,
+  query as firestoreQuery
+} from "firebase/firestore";
+import {
+  FirestoreState,
+  type FirestoreStateOptions,
+  type PathParam
+} from "./FirestoreState.svelte.js";
+
+type CollectionAggregateStateOptions = Omit<
+  FirestoreStateOptions<any, any>,
+  "pathFunctionOrString" | "fromFirestore" | "toFirestore" | "listen"
```

Comment:
Necessary to remove `fromFirestore` and `toFirestore`? I don't have a use-case, but it does add some variation to the API depending on which State you're using.

Reply:
This class should not set or get document data so I don't really see a use case either for now.